framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -8890,6 +8990,17 @@ void VulkanReplayConsumerBase::UpdateDescriptorSetInfoWithTemplate(DescriptorSet
     }
 }
 
+VulkanBufferTracker& VulkanReplayConsumerBase::GetBufferTracker(VkDevice device)
+{
+    auto it = _buffer_trackers.find(device);
+    if (it == _buffer_trackers.end())
+    {
+        auto [new_it, success] = _buffer_trackers.insert({ device, VulkanBufferTracker(object_info_table_) });
```

Comment:
What is the ", success" part doing?  It's not used later so if it's receiving something, it's wasted.

Reply:
yes, true, we don't really need it. guess was just drinking the structured bind cool-aid to assign variable-names (vs. first/second from std::pair) to the return pair (and normally no compiler-warnings for unused elements). will change

Reply:
the success-value would be used to check if we actually inserted something (or if an element was already present for that key). but that's obsolete here as you point out

Reply:
@bradgrantham-lunarg mentioned this isn't a big deal.  So don't worry about it.